.asciidoctor/lib/const-inline-macro/extension.rb

```diff
@@ -49,8 +49,8 @@ def process parent, target, attrs
       end
     end
     if const_value[0][0].nil?
-        missing_attr = if attrs.has_key? 'tag' then attrs['tag'] else attrs['name'] end
-        raise "Failed reading '#{data_path}'. Check if constant '#{missing_attr}' exists"
+      missing_attr = if attrs.has_key? 'tag' then attrs['tag'] else attrs['name'] end
```

Comment:
This can be simplified as `missing_attr =  attrs.has_key? 'tag' ? attrs['tag'] : attrs['name'] `

Reply:
I'm not sure, as IRB gives me this:

```ruby
2.4.1 :034 > attrs = {}
 => {} 
2.4.1 :035 > attrs['tag'] = "test"
 => "test" 
2.4.1 :036 > missing_attr = attrs.has_key? 'tag' ? attrs['tag'] : attrs['name']
(irb):36: warning: string literal in condition
 => false 
2.4.1 :037 > puts missing_attr
false
 => nil 
2.4.1 :038 > 
```

Reply:
`missing_attr = (attrs.has_key? 'tag') ? attrs['tag'] : attrs['name']` would do